eogrow/core/schemas.py

```diff
@@ -24,25 +25,25 @@
 class ManagerSchema(BaseSchema):
     """A basic schema for managers, to be used as a parent class for defining manager schemas"""
 
-    manager: Optional[ImportPath] = Field(description="An import path to this specific manager.")
+    manager: Optional[ImportPath] = Field(None, description="An import path to this specific manager.")
```

Comment:
are the new type hints not applied to field types in this case? or are we not using that here?

Reply:
you mean the new-style syntax of `str | None`? won't work until 3.10, 'future annotations' backports it but only during typechecking.

Reply:
I guess we could increase it to 6, since it's usually between 6 and 7 digits, but I went with 5 to allow some more numerical differences